index.html

```diff
@@ -1298,6 +1299,15 @@ <h4>Create Exchange</h4>
           data-api-endpoint="post /workflows/{localWorkflowId}/exchanges"></div>
       </section>
 
+      <section>
+        <h4>Get Exchange Protocols</h4>
+        <p>
+        </p>
+
```

Comment:
Maybe we need to explain how `interact` is supposed to work in this section?

Reply:
Or link to the interaction section? This section should say that the returned object lists all of the protocols supported by the exchange -- and that an implementation may implement an interaction URL handler by proxying/internally fetching this endpoint and returning its result to the caller.

We could use a mermaid style flowchart here to show an interaction URL being retrieved from a coordinator that internally looks up the associated exchange and fetches `<exchangeId>/protocols` and returns it to the caller. We can then explain that this allows the coordinator to delegate to an exchange service whereby the caller (client, e.g., digital wallet) can still use their trust in the coordinator origin -- as it serves (passes through) the `protocols` object.

Reply:
Fixed in ef7fac95c54552b531b89d39abe03da50f29ea9a.